### PR TITLE
Updating the strings

### PR DESCRIPTION
**Pull request by nethip**
As per suggestions from @peterFlynn and @MarcelGerber

### CODE REVIEW

**Comment by nethip** [comment]
@peterflynn @MarcelGerber Please review this and merge if it looks good.

File: updates/stable/en.json [hunk 1]
```diff
@@ -8,22 +8,22 @@
...
+                "description": "Extensions can now be individually enabled/disabled from the Extension Manager."
             },
             {
                 "name": "Improved Text Rendering on Mac",
-                "description": "Crisp text rendering on Mac with subpixel antialiasing enabled by default. You can use the older text rendering by setting the `fonts.fontSmoothing` preference to `antialiasing`."
+                "description": "Crisp text rendering on Mac with subpixel antialiasing enabled by default. You can use the older text rendering by setting the `fonts.fontSmoothing` preference to `antialiased`."
             },
             {
-                "name": "Greek and Cyrillic character set support",
+                "name": "Greek and Cyrillic Character support",
```
Comment (peterflynn) [inline updates/stable/en.json]: "Support" still needs to be uppercase

File: updates/stable/en.json [hunk 2]
```diff
@@ -8,22 +8,22 @@
...
             {
                 "name": "Instant Search in Files",
-                "description": "Searching across files has never been this faster. The search results are now displayed as and when you type in the search box."
+                "description": "Searching across files is faster than ever. Results are now displayed incrementally and updated as you type."
             },
             {
-                "name": "Improved Preferences Experience",
-                "description": "All preferences are now available as code hints while editing the `brackets.json` preferences file. Opening the Preferences Files will also open the default preferences in a split view for reference."
+                "name": "Easier Preferences Editing",
+                "description": "All preferences are now available as code hints while editing the `brackets.json` preferences file. Opening a preferences file will also open the default preferences in a second pane."
```
Comment (peterflynn) [inline updates/stable/en.json]: I think it'd be better to keep "for reference" at the end, but I don't feel strongly

**Comment by peterflynn** [comment]
@nethip Just two small pieces of feedback, for my part

**Comment by nethip** [comment]
@peterflynn Done with change. Can you merge this

**Comment by peterflynn** [comment]
Thanks for the updates!

**Comment by nethip** [comment]
Thanks @peterflynn and @MarcelGerber for the suggestions.

**Pull request merged**